server/ladder_service.py

```diff
@@ -133,10 +134,13 @@ async def fetch_map_pools(self, conn) -> Dict[int, Tuple[str, List[Map]]]:
                 )
             elif row.map_params is not None:
                 try:
-                    params = json.loads(row.map_params)
+                    params = row.map_params
                     map_type = params["type"]
                     if map_type == "neroxis":
-                        map_list.append(NeroxisGeneratedMap.of(params, row.weight))
+                        map_list.append(NeroxisGeneratedMap.of(
+                            params,
+                            row.weight
+                        ))
```

Comment:
This might be better
```suggestion
                        map_list.append(
                            NeroxisGeneratedMap.of(params, row.weight)
                        )
```